src/components/Tokens/TokenDetails/MobileBalanceSummaryFooter.tsx

```diff
@@ -82,7 +82,8 @@ export default function MobileBalanceSummaryFooter({
   tokenAmount,
   nativeCurrencyAmount,
   isNative,
-}: BalanceSummaryProps) {
+  tokenAddress,
+}: BalanceSummaryProps & { tokenAddress?: string }) {
```

Comment:
you should already have `tokenAddress` available on `tokenAmount`

Reply:
good pt, fixed

Reply:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/5773490/195405587-253b22db-afc4-45a8-bc73-74a297a32ba6.png">

it appears to be working in the case I just tested... 🤔

Reply:
<img width="915" alt="image" src="https://user-images.githubusercontent.com/5773490/195405870-898e178c-5662-4e03-9963-b330b9f4f93c.png">

I've tested: (on ethereum w/ a connected wallet) ETH ("NATIVE"), WETH, USDT (no balance), DAI (w/ balance) and all worked as expected

Reply:
hmmmmm